src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -677,6 +719,11 @@ public void onUpdate() {
                     player.addPotionEffect(new PotionEffect(ModPotions.thirst, hunterEffect.getDuration(), hunterEffect.getAmplifier()));
                     player.removePotionEffect(MobEffects.HUNGER);
                 }
+                if (player.ticksExisted % 10 == 0) {
```

Comment:
We don't want to sync the victim id twice per second.
It should only be sent if it changes.

For ending the bite process, you could use the exiting `syncPacket` here, by writing the id to the exiting object and setting `sync` to true.

Reply:
fixed as of latest commit.